### PR TITLE
Chemical synthesis kit tweak

### PR DESCRIPTION
**Pull request by ThatGuyUSA**
## About the PR
Changed the label in the hyperzine syringe given in the chemical synthesis kit

## Why
This looked like it was named through coding naming conventions

## Technical details
Line 1039 in 'healing.yml' Label component was changed from "reagent-name-hyperzine" to "hyperzine"

## Media
OLD
image

NEW
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

### CODE REVIEW

**Comment by ThatGuyUSA** [comment]
guys how do u separate commits from other projects that you were working on but then actually decided to stop working on to instead work on something that you could feasibly do

**Comment by deltanedas** [comment]
stash changes with git stash

**Review by UbaserB — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 1]
```diff
@@ -1036,7 +1036,7 @@
   id: SyringeStimulants
   components:
   - type: Label
-    currentLabel: reagent-name-hyperzine
+    currentLabel: hyperzine
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:1039]:
```suggestion
    currentLabel: reagent-name-stimulants
```
You are removing the localization and hardcoding the english name, which is bad for forks in other languages.
Use the correct localization string instead. Stimulants got renamed into hyperzine at some point, but the loc string stayed the same.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml (hunk 1 above)
Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:1039]: but all the other things in the file use the same system, does it still use the localisation there?

Comment (slarticodefast) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:1039]: The loc string was changed, but the ID is still the same as before
grafik

Comment (ThatGuyUSA) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:1039]: as long as it gets fixed! how long has it been like this anyways?

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**